.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
out of curiosity, how does this get included if the gitignore is supposed to ignore the `.vscode/` directory?

Reply:
That's an error. It should be excluded as it's personal preference - thought it was in .gitnore. I'll remove.

Reply:
removed